### ISSUE TITLE
As an admin I need to assign a Baseline Volunteer to a Gene

### ISSUE THREAD

**Issue by yugen** [clingen-data-model/community-curation#20]
As an admin I want to assign a Baseline volunteer need to a gene.

Acceptance Criteria:
- Admin should only be able to assign genes to volunteers that have Basic Evidence Aptitude
- Admin should be able to CRUD genes

Punch List:
- [x] Create a gene model that implements AssignableContract
- [x] Add scopeGene to Assignment model
- [x] Add isGene to AssignmentCollection
- [x] Add admin interface to add/update genes
- [x] Seed genes table with data
- [x] Create UI to assign Baseline volunteer to Gene
- [x] User should have access to the gene's protocol, if any, when assigned to a gene.
- [x] Set volunteer status to 'active' if not already active.

**Comment by yugen** [comment]
@courtneythaxton @lizzkearns. You should now be able to create and assign a gene to a baseline volunteer who is trained and attested.

**To create a gene:** 
1. Log in as an admin or programmer and go to http://community-curation.sirsdemo.unc.edu/admin/gene to see a list of genes in the system.
2. Click the Add Gene button to create a new gene.

**To assign a gene:**
1. Go to a baseline volunteer's detail screen
2. Click the 'Manage Assignments' button
3. Mark the volunteer trained and complete their attestation.
4. return to their detail screen and click 'Manage Assignments'
5. Click the 'Add Gene' button, select a gene, and click save.
6. The gene should now be listed and, if set, a link to the protocol will appear.

**Comment by courtneythaxton** [comment]
@yugen I tested the functionality first for adding a new gene. 

I was able to add MYBPC3, entered the HGNC ID (7551), as a assume that is what is needed when Hgnc, is referred too? It might be good to call it HGNC ID? 

I was also able to upload a protocol for MYBPC3.

For hypothes.is group, will this hold the name of the group only, or can this actually hold the link that allows a member to join the group? 

**Action Item:** IT might be good to also have the link to join, as this can either be sent in an email or indicated on the CCDB volunteers page.

**Bug?:** Once I saved the record, I noticed that the "protocol filename did NOT show up on the Genes page afterwards, even though I added a file?

I performed steps one through 6 which worked well. However, when I tried to access the protocol hyperlink, it gave me a "404/ Not found" error.

**Comment by yugen** [comment]
@courtneythaxton 
- I've added hypothesis_group_url to genes so you can now add those.  If you add a hypothesis group the form will require a hypothesis group url.
- I am currently able to upload protocol files for genes without a problem.  Let me know if the issue persists.
- You should now be able to download a protocol file.

**Comment by courtneythaxton** [comment]
@yugen 

I tested out the ability to add a gene and a protocol. I was able to upload a protocol, and added a Hypothesis group name and URL. I attempted to add only the numerals for the HGNC ID, and was alerted that the HGNC:XXXX was required, so your alert works.

I was able to download the hyperlinked protocol.

While impersonating the user (rekha aaron), I was also able to download the protocol for MYH7.

I was not able to see as either the admin, or the user the hypothesis.is annotation group url or name.
